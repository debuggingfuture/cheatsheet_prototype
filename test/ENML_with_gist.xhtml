<!DOCTYPE en-note SYSTEM "http://xml.evernote.com/pub/enml2.dtd"><en-note style="word-wrap: break-word; -webkit-nbsp-mode: space; -webkit-line-break: after-white-space;"><gist user='vincentlaucy' id='5548010' /><div>Test123</div><gist user='vincentlaucy' id='123' /><div>Test2</div></en-note>