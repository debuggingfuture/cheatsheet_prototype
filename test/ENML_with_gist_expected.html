<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body style="word-wrap: break-word; -webkit-nbsp-mode: space; -webkit-line-break: after-white-space;"><link href="https://gist.github.com/assets/embed-17ab34a51711628d8f5449c4663a9318.css" media="screen" rel="stylesheet"><div id="gist5548010" class="gist">      <div class="gist-file">        <div class="gist-data gist-syntax">  <div class="file-data">    <table cellpadding="0" cellspacing="0" class="lines highlight">      <tbody><tr>        <td class="line-numbers">          <span class="line-number" id="file-gistfile1-js-L1" rel="file-gistfile1-js-L1">1</span>        </td>        <td class="line-data">          <pre class="line-pre"><div class="line" id="file-gistfile1-js-LC1"><span class="nb">document</span><span class="p">.</span><span class="nx">write</span><span class="p">(</span><span class="s1">'test'</span><span class="p">);</span></div></pre>        </td>      </tr>    </tbody></table>  </div>        </div>        <div class="gist-meta">          <a href="https://gist.github.com/vincentlaucy/5548010/raw/094a3e07ef9196894df510e6c1b7add8c3ae3c64/gistfile1.js" style="float:right">view raw</a>          <a href="https://gist.github.com/vincentlaucy/5548010#file-gistfile1-js" style="float:right; margin-right:10px; color:#666;">gistfile1.js</a>          <a href="https://gist.github.com/vincentlaucy/5548010">This Gist</a> brought to you by <a href="http://github.com">GitHub</a>.        </div>      </div></div><div>Test123</div><gist user="vincentlaucy" id="123"><div>Test2</div><div><em>Sorry This Gist render failed!</em></div><div>Test2</div></gist></body></html>